### ISSUE TITLE
It seems not working anymore

### ISSUE THREAD

**Issue by ofou** [ActivityWatch/aw-import-screentime#3]
When I ran `poetry install` it says 

```
Installing dependencies from lock file

Could not parse version constraint: branch
```

Any ideas?

**Comment by abegong** [comment]
I get the same error. I was able to get the `main.py` script to run by pip installing `aw_core` and `aw_client`, but then it crashes with:

```
Reading from database file at /Users/abe/Library/Application Support/Knowledge/knowledgeC.db
Sending events to ActivityWatch...
Traceback (most recent call last):
  File "main.py", line 111, in <module>
    main()
  File "main.py", line 75, in main
    send_to_activitywatch(events)
  File "main.py", line 96, in send_to_activitywatch
    aw.send_events(bucket, events)
AttributeError: 'ActivityWatchClient' object has no attribute 'send_events'
```

I've inspected the query and data that it returns. It looks like those are all working fine, and it's just a problem with `ActivityWatchClient`. Maybe the object's APIs have been refactored over the last couple of years...?

**Comment by abegong** [comment]
Ha. I was right. The method is now called `insert_events` instead of `send_events`. I'll submit a PR to fix this.

**Comment by ErikBjare** [comment]
Fixed in #4
